endpoint-sec-sys/src/types.rs

```diff
@@ -343,6 +343,10 @@ ffi_wrap_enum!(
     == MACOS_15_0_0;
     --
     ES_EVENT_TYPE_NOTIFY_GATEKEEPER_USER_OVERRIDE = 146,
+
+    == #[cfg(feature = "macos_15_4_0")] 15_4_0 "15.4.0";
```

Comment:
Shouldn't it be instead:

```suggestion
    == MACOS_15_4_0;
```

like the above?